app/views/admin/statistics_announcements/show.html.erb

```diff
@@ -69,6 +69,33 @@
         <%= link_to "Cancel statistics release", [:cancel, :admin, @statistics_announcement], class: "btn btn-default pull-right" %>
       <% end %>
     </div>
+
+    <% if @statistics_announcement.can_be_tagged_to_taxonomy? %>
+      <section class="taxonomy-topics" id="topic-new-taxonomy">
```

Comment:
this is quite a complex bit of view code. Could it be made into a partial? How much does it have in common with the `edition` version?